src/GslibIO.jl

```diff
@@ -44,6 +55,23 @@ function load(file::File{format"GSLIB"})
   end
 end
 
+# helper function to parse a legacy GSLIB file
+function parse_legacy(filename::AbstractString)
+  open(filename) do fs
+    header = readline(fs)
```

Comment:
You can either add a comment before this line saying something trivial like "first line is a free header" or remove the comments of the following sections for consistency. Also, please add a blank line after this `header` line so that it is clear that there are 3 sections you are reading from the file: header, varnames, data.